src/pages/home/_why-deriv-mobile.js

```diff
@@ -96,7 +96,7 @@ const WhyDerivMobile = () => {
                 </CardItem>
             </Card>
             <ButtonWrapper>
-                <SignupButton secondary to="/signup/">
+                <SignupButton secondary="true" to="/signup/">
```

Comment:
Is it mandatory to set the value to `true`?

Reply:
@sara-fs for styled-components props yes